### PR TITLE
Add heroku CLI.

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

File: heroku/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Heroku CLI snap
+
+This project creates a working snap for Heroku's CLI.
+Heroku is migrating it's CLI to go, so this project is evolvint rapidly. Getting
```
Comment (dplanella) [inline heroku/README.md]: * evolving

File: heroku/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+it packaged in a snap and installable in ubuntu from edge, beta and stable
```
Comment (dplanella) [inline heroku/README.md]: Ubuntu (starting with caps)

**Comment by elopio** [comment]
I don't know what happened. I'll make the corrections and propose it again without plank.

**Comment by dholbach** [comment]
It was a commit by Mike which we reverted in master.

**Comment by elopio** [comment]
Thanks for the review. The typos are fixed and I rebased, so now the diff looks good.

**Comment by dholbach** [comment]
Thanks a lot! Good work! :-)

**Pull request merged**